docs/ref/models/conditional-expressions.txt

```diff
@@ -236,3 +236,18 @@ On other databases, this is emulated using a ``CASE`` statement:
 
 The two SQL statements are functionally equivalent but the more explicit
 ``FILTER`` may perform better.
+
+Conditional filter
+------------------
+
+When a conditional expression returns a boolean value, it is possible to
+use the expression directly in a filter expression. This means it will not
+be returned as a column in the query, but will still be used to filter
+results::
+
+    >>> Client.objects.filter(Case(
+    ...     When(account_type=Client.GOLD, then=Value(True)),
+    ...     When(account_type=Client.PLATINUM, then=Value(True)),
+    ...     default=Value(False),
+    ...     output_field=BooleanField()
+    ... ))
```

Comment:
I'd be great to find a better example as this is a really verbose `filter(Q(account_type=Client.GOLD)|Q(account_type=Client.PLATINUM))` or even `filter(acount_type__in=[account_type=Client.GOLD, account_type=Client.PLATINUM])`.

What about

```python
Client.objects.filter(
    ~Exists(Clients.objects.exclude(pk=OuterRef('pk')).filter(account_type=OuterRef('account_type')),
)
```

Which results in

```sql
SELECT *
FROM client c
WHERE NOT EXISTS(
    SELECT 1 FROM client i
    WHERE i.account_type = c.account_type AND NOT i.id = c.id
)
```

And returns all the clients that are uniquely using an `account_type`?

Reply:
Yeah, that's a nice one.